packages/fury/test/fury-test.js

```diff
@@ -409,22 +409,24 @@ describe('Fury class', () => {
   describe('pass mediaType into adapter operations', () => {
     const fury = new Fury();
 
-    function test(first) {
-      return function impl(second) { assert.equal(first, second); };
+    function test(expectedMediaType) {
```

Comment:
This test took me a while to get my head around it so I tried to re-name some of the parts to make it simpler. This test also was throwing a string which is tricky because we can't reject a promise with a string (it shows a warning when you do that), it's probably not a good idea to do anyway.

I've updated so we return a parse result containing the media type instead.